core/domain/topic_domain.py

```diff
@@ -93,7 +95,11 @@ def __init__(self, change_dict):
         elif self.cmd == CMD_REMOVE_UNCATEGORIZED_SKILL_ID:
             self.id = change_dict['uncategorized_skill_id']
         elif self.cmd == CMD_MOVE_SKILL_ID_TO_SUBTOPIC:
-            self.id = change_dict['subtopic_id']
+            self.old_subtopic_id = change_dict['old_subtopic_id']
+            self.new_subtopic_id = change_dict['new_subtopic_id']
+            self.skill_id = change_dict['skill_id']
+        elif self.cmd == CMD_REMOVE_SKILL_FROM_SUBTOPIC:
```

Comment:
Either MOVE_SKILL_TO_SUBTOPIC and REMOVE_SKILL_FROM_SUBTOPIC

or

MOVE_SKILL_ID_TO_SUBTOPIC and REMOVE_SKILL_ID_FROM_SUBTOPIC

but not a mix of both. Be consistent with other cmds too.

Reply:
Done, changed to REMOVE_SKILL_ID_FROM_SUBTOPIC as other commands used skill_id.